kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/TopicEnforcer.java

```diff
@@ -88,12 +94,18 @@ public TopicEnforcer(
             .filter(
                 t -> {
                   Result result = configDrift.check(t, existing.get(t.getName()), type);
-                  if (logResults && !Result.NO_DRIFT.equals(result)) {
-                    LOG.info(
-                        "Found {} for topic {}, in {} drift detection mode",
-                        result,
-                        t.getName(),
-                        type);
+                  if (logResults) {
+                    String msgFmt = "Found {} for topic {}, in {} drift detection mode";
+                    switch(result) {
+                      case NO_DRIFT:
+                        break;
+                      case SAFE_DRIFT:
+                        LOG.info(msgFmt, result, t.getName(), type);
+                        break;
+                      default:
```

Comment:
would be good to explicitly call out the UNSAFE enum here, for readability sake

Reply:
refactored this, let me know what you think